webnode/src/redux/epics/node-epics.js

```diff
@@ -211,14 +209,20 @@ const checkIfSectorClaimedEpic = (action$, store) => {
       const specialChunkIdx = sectorIdx * CHUNKS_PER_SECTOR;
       const dataMap = Datamap.rawGenerate(genesisHash, numberOfChunks);
       const dataMapHash = dataMap[specialChunkIdx];
-      const address = Datamap.obfuscate(dataMapHash);
+
+      const address = iota.toAddress(
+        iota.utils.toTrytes(Encryption.obfuscate(dataMapHash))
+      );
       // const address =
       //   "HT9MZQXKVBVT9AYVTISCLELYWXTILJDIMHFQRGS9YIJUIRSSNRZFIZCHYHQHKZIPGYYCSUSARFNSXD9UY";
 
       return Observable.fromPromise(
         iota.findMostRecentTransaction(address)
       ).map(transaction => {
-        if (iota.checkIfClaimed(transaction)) {
+        if (
+          iota.checkIfClaimed(transaction) &&
+          !TEST_GENESIS_HASHES.includes(genesisHash)
```

Comment:
don't usually like doing this since it is using test code in real code. but this is to force it to disregard whether "the sector was claimed" in dev mode.

Reply:
I see what you mean.  Before, when you were saying we couldn't use one more than once, I thought you were talking about because the treasure would have already been claimed.